vendor/wasm/js/runtime.js

```diff
@@ -63,9 +63,8 @@ class WasmMemoryInterface {
 		return lo + hi*4294967296;
 	};
 	loadI64(addr) {
-		// TODO(bill): loadI64 correctly
 		const lo = this.mem.getUint32(addr + 0, true);
-		const hi = this.mem.getUint32(addr + 4, true);
+		const hi = this.mem.getInt32 (addr + 4, true);
```

Comment:
Maybe remove this space before open paren?

Reply:
I was trying to follow the existing style, you can see similar aligning here: https://github.com/odin-lang/Odin/blob/master/vendor/wasm/js/runtime.js#L102-L105